mlrun/api/crud/model_endpoints.py

```diff
@@ -866,16 +875,31 @@ def deploy_model_monitoring_batch_processing(
         task = mlrun.new_task(name="model-monitoring-batch", project=project)
         task.spec.function = function_uri
 
+        # Apply batching interval params
+        def _parse_intervals_from_cron_format(element: str):
+            # Get the interval value from the cron time format
+            res = element.partition("/")[-1].strip(" ")
+            return 0 if res == "" else float(res)
+
+        minutes, hours, day, _, _ = tuple(
+            map(
+                _parse_intervals_from_cron_format,
+                tracking_policy["default_batch_intervals"].split(" "),
+            )
+        )
+        batch_dict = {"minutes": minutes, "hours": hours, "days": day}
+        task.spec.parameters["batch_intervals_dict"] = batch_dict
```

Comment:
There should be a constant variable for keys like `"batch_intervals_dict"`. 

Edit: I see now there is at model_monitoring/constants so import it and use it :)